### DISCUSSION TITLE
Reflect new account names in the dashboard

### DISCUSSION THREAD

**Discussion by MrAtheist** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#137] [Q&A] [answered]
After the dashboard was created, it doesnt seem like the drop down for `Account Names` is reflected with accounts that are created and/or invited into the org. On the other hand, `Linked Account IDs` seems to have the entire list for the org. Is there a way to achieve the same thing for `Account Names`..? I spun up with the legacy code and installed v4.60.2

<img width="948" alt="image" src="https://user-images.githubusercontent.com/4513916/148666074-8c3dd154-19b0-4039-93f8-137d34238c8a.png">

**Comment by darken99** [comment, marked as answer]
As of v0.1.7 tool supports [Organization Data CUR Connection](https://www.wellarchitectedlabs.com/cost/300_labs/300_organization_data_cur_connection/)
You'll have to create organization data table in the same Athena database in order the tool to detect it.
Once completed the lab run `cid-cmd map` and it should recreate account maps using that table.
